.circleci/config.yml

```
@@ -47,16 +47,24 @@ jobs:
           root: .
           paths:
             - .
-  lint_backend_and_frontend:
+  lint_backend:
```

```suggestion
  lint_backend_and_frontend:
```
We'd leave this job name alone.